include/csoundCore.h

```diff
@@ -203,11 +203,11 @@ extern "C" {
 #define CURTIME (((double)csound->icurTime)/((double)csound->esr))
 #define CURTIME_inc (((double)csound->ksmps)/((double)csound->esr))
 
-#ifdef  B64BIT
-#define MAXLEN     0x10000000
-#define FMAXLEN    ((MYFLT)(MAXLEN))
-#define PHMASK     0x0fffffff
-#else
+#ifndef  SHORT_TABLE_LENGTH  // long max table length is the default
+static const uint32_t MAXLEN = 1U << 31;  
+static const double FMAXLEN = (double) (1U << 31);
+static const uint32_t PHMASK = (1U << 31) - 1U;
+#else   // this is the original max table length
 #define MAXLEN     0x1000000L
```

Comment:
shouldn't this branch also use static const definitions?

Reply:
I wanted to preserve the original code as it was (mostly for archaeology reasons). Not worried if you feel we ought to change it.

Reply:
With the #ifdef it implies we are going to support SHORT_TABLE_LENGTH. If the following is there just for historic purposes, then I'd suggest removing the #ifdef and just leaving the original code commented out.

Reply:
I think there's a dual purpose, documenting and supporting (maybe some systems may need this? Although I think the only purpose is to make it compatible with single-float precision). I can change it to consts if you wish.